src/xdocs/config_coding.xml

```diff
@@ -2163,14 +2175,29 @@ try {
 &lt;module name=&quot;IllegalThrows&quot;&gt;
   &lt;property name=&quot;illegalClassNames&quot; value=&quot;NullPointerException&quot;/&gt;
 &lt;/module&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+public class Test{
+    public void func1() throws RuntimeException{} // ok
+
+    public void func2() throws NullPointerException{} // violation
+}
         </source>
         <p>
-          To configure the check ignoring method named &quot;foo()&quot;:
+          To configure the check ignoring method named &quot;func1()&quot;:
         </p>
         <source>
 &lt;module name=&quot;IllegalThrows&quot;&gt;
-  &lt;property name=&quot;ignoredMethodNames&quot; value=&quot;foo&quot;/&gt;
+  &lt;property name=&quot;ignoredMethodNames&quot; value=&quot;func1&quot;/&gt;
 &lt;/module&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+public class Test{
+    public void func1() throws RuntimeException{}  // ok
+    public void func2() throws Error{}  // violation
+}
```

Comment:
Space before brace(s) missing: ` {`. Please use the same style as the other examples.

Reply:
Done